src/Language/PureScript/TypeChecker/Monad.hs

```diff
@@ -310,26 +310,15 @@ putEnv env = modify (\s -> s { checkEnv = env })
 modifyEnv :: (MonadState CheckState m) => (Environment -> Environment) -> m ()
 modifyEnv f = modify (\s -> s { checkEnv = f (checkEnv s) })
 
--- | Run a computation in the typechecking monad, starting with an empty @Environment@
-runCheck :: (Functor m) => StateT CheckState m a -> m (a, Environment)
```

Comment:
I can imagine that this might be useful for development or testing? I don't think I've ever used it myself though.

Reply:
I doubt it's even useful for that. Currently the ‘typechecking monad’ is a combination of `MonadState`, `MonadWriter`, and `MonadError`; this is just a flimsy wrapper around `runStateT`.

Reply:
Fair enough - I'm happy to remove this then.